src/cve/getLocalCommits.py

```diff
@@ -0,0 +1,73 @@
+import json
+import os
+import subprocess
+import shutil
+
+def check_commit_exists(repo_path, commit_hash):
+    """Check if a commit exists in the specified repository using git cat-file."""
+    try:
+        # Debug: Print the command being run
+        #print(f"Running: git -C {repo_path} cat-file -e {commit_hash}")
+
+        # Run git cat-file to check if the commit exists
+        result = subprocess.run(
+            ["git", "-C", repo_path, "cat-file", "commit", commit_hash],
+            check=True,
+            stdout=subprocess.PIPE,
+            stderr=subprocess.PIPE,
+        )
+        print(f"Success: Commit {commit_hash} found in {repo_path}")
+        return True
+    except subprocess.CalledProcessError as e:
+        # Debug: Print the error message
+        #print(f"Error: {e.stderr.decode('utf-8')}")
+        #print(f"Commit {commit_hash} not found in {repo_path}")
+        return False
+
+def main():
+    #debugging
+    print(shutil.which("git"))
+
+    # File containing patches
+    json_file = "nvdcve-github-patches.json"
```

Comment:
Might be good to add a comment here on how this file was generated, e.g. what script generated that file.